src/components/Shell/ShellAppBar.tsx

```diff
@@ -95,30 +100,40 @@ export const ShellAppBar = ({
         >
           {title}
         </Typography>
-        <Tooltip title="Copy current URL">
-          <IconButton
-            size="large"
-            edge="start"
-            color="inherit"
-            aria-label="Copy current URL"
-            sx={{ ml: 'auto' }}
-            onClick={onLinkButtonClick}
-          >
-            <LinkIcon />
-          </IconButton>
-        </Tooltip>
         {doShowPeers ? (
-          <Tooltip title="Click to show peer list">
-            <IconButton
-              size="large"
-              edge="end"
-              color="inherit"
-              aria-label="Peer list"
-              onClick={onPeerListClick}
-            >
-              <StepIcon icon={numberOfPeers} />
-            </IconButton>
-          </Tooltip>
+          <Box display="inline-flex">
+            <Tooltip title="Open QR Code">
```

Comment:
I'd like to tweak how this copy flows a bit:

```suggestion
            <Tooltip title="Show QR Code">
```

Reply:
Addressed